src/SourceGenerators/Uno.UI.SourceGenerators/Uno.UI.SourceGenerators.csproj

```diff
@@ -108,6 +108,12 @@
 		<None Include="Content\Uno.UI.SourceGenerators.props" />
 	</ItemGroup>
 
+	<ItemGroup>
+		<InternalsVisibleTo Include="SourceGeneratorsConsole" />
```

Comment:
@jeromelaban Was this a thing at some point that is now deleted?

Reply:
Indeed it was, it has been removed a long time ago.

Reply:
```suggestion
```